example_11/doc/userdoc.rst

```diff
@@ -96,6 +96,8 @@ Tutorial steps
    .. code-block:: shell
 
     ros2 topic pub --rate 30 /bicycle_steering_controller/reference geometry_msgs/msg/TwistStamped "
+      header:
+        stamp: now
```

Comment:
```suggestion
      header: auto
```
I wasn't aware of this, but for `std_msgs/msg/Header` this should also work (from humble on)
https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Topics/Understanding-ROS2-Topics.html#ros2-topic-pub